l10n_it_fiscalcode/model/res_partner.py

```diff
@@ -1,31 +1,55 @@
 # -*- coding: utf-8 -*-
 # Copyright 2014 Associazione Odoo Italia (<http://www.odoo-italia.org>)
 # Copyright 2016 Andrea Gallina (Apulia Software)
+# Copyright 2016 Giuliano Lotta
+# version 0.1- changed method check_fiscalcode() to consider multiple situation
```

Comment:
The history of the module is in the commits. We don't need of a description of every version.